assets/scripts/whichx.js

```diff
@@ -158,18 +201,29 @@ function WhichX(config) {
         return typeChance * (type.tcount / total.tcount);
     }
 
-    // A non-zero prior estimate to prevent 0 based probability calculations.
+    /**
```

Comment:
This function is not available to call on the `WhichX` object, so the JSDoc here may not be all the helpful. Can we remove the docs to keep file size down?

Reply:
See above comment